### PR TITLE
feat: add frontend to go to next logo

### PR DESCRIPTION
**Pull request by caroso1222**
@edgargonzalez525, I've configured everything in the front except styling. I'll just need a few more minutes on Monday to polish and we're good to go.

Also, please note I'm considering `nextLogo` to be an object rather than just the ID. This will allow us to scale and send more data in the future like the "obfuscatedImageUrl" in case we want to show the user a sneak peek of the next logo.

### CODE REVIEW

File: apps/logo-quiz/src/main.tsx [hunk 1]
```diff
@@ -20,35 +20,38 @@ import { FirebaseContext } from './shared/components/firebase/with-firebase';
 import { Firebase } from './shared/components/firebase/firebase';
 import { ROUTES } from './shared/utils/routes';
 import { environment } from '@logo-quiz/environment';
+import { ErrorBoundary } from './shared/components/error-boundary/error-boundary';
 
 const loggerMiddleware = createLogger();
 
 const middlewares = [
-  thunkMiddleware,
+  thunkMiddleware, //
```
Comment (edgargonzalez525) [inline apps/logo-quiz/src/main.tsx:28]: are the // necessary?

File: apps/logo-quiz/src/main.tsx [hunk 2]
```diff
@@ -20,35 +20,38 @@ import { FirebaseContext } from './shared/components/firebase/with-firebase';
...
   environment.production ? null : loggerMiddleware,
 ].filter(Boolean);
 
 const store = createStore(
-  rootReducer,
+  rootReducer, //
```
Comment (edgargonzalez525) [inline apps/logo-quiz/src/main.tsx:33]: are the // necessary?

File: apps/logo-quiz/src/store/logo/types.ts [hunk 3]
```diff
@@ -36,6 +36,7 @@ export interface LogoState {
   isVerifying: boolean;
   status: LogoStatus;
   realImageUrl: string;
+  nextLogo?: Partial<Pick<Logo, 'id' | 'obfuscatedImageUrl'>>; // TODO: maybe it's worth it to create a new type NextLogo with this type definition?
```
Comment (edgargonzalez525) [inline apps/logo-quiz/src/store/logo/types.ts:39]: it should be _id and not id

File: apps/logo-quiz/src/main.tsx (hunk 1 above)
Comment (caroso1222) [reply apps/logo-quiz/src/main.tsx:28]: Yeah. I'm using prettier and [it requires that nasty hack](https://github.com/prettier/prettier-vscode/issues/352#issuecomment-417471927) to put everything into a single line. It sucks, I know :/

File: apps/logo-quiz/src/main.tsx (hunk 2 above)
Comment (caroso1222) [reply apps/logo-quiz/src/main.tsx:33]: See comment above ^

File: apps/logo-quiz/src/main.tsx (hunk 1 above)
Comment (edgargonzalez525) [reply apps/logo-quiz/src/main.tsx:28]: alright, good to know about this

**Pull request merged**